src/common/routes.js

```diff
@@ -261,16 +267,36 @@ const routes = [
             exact: true,
             component: PaymentCampaign,
           },
+          {
+            path: '/rewards/payables/@:userName/:reservationPermlink',
+            exact: true,
+            component: PaymentCampaign,
+          },
           {
             path: '/rewards/receivables/@:userName',
             exact: true,
             component: PaymentCampaign,
           },
+          {
+            path: '/rewards/receivables/@:userName/:reservationPermlink',
```

Comment:
we can union routes: 
'/rewards/receivables/@:userName' + '/rewards/receivables/@:userName/:reservationPermlink' => 
=> '/rewards/receivables/@:userName/:reservationPermlink?'